resources/Materials/Examples/Units/distance_units.mtlx

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These new materials should probably be moved to Materials/TestSuite, since they're more focused on testing MaterialX features, where Materials/Examples is more focused on user-facing material examples.